### ISSUE TITLE
name suggestions

### ISSUE THREAD

**Issue by kosack** [cta-observatory/configsystem#5]
A few name ideas, since that's still on the open issue list...

* **configurist** (a figurest = one who uses figures/numbers)
* **configeater** (a figeater is an iridescent green beetle, so obvious logo)
* **configment** (figment = imaginary, related to imagination)

**Comment by maxnoe** [comment]
To be totally honest... I not that big of a fan of "funny" / "creative" names anymore, I'd much rather just call this "cta_configsystem"...

**Comment by kosack** [comment]
since it's more general than CTA (which is also now CTAO), you might not want to add CTA, as it might get used outside later if it's nice

**Comment by kosack** [comment]
Too generic a name is also not great though, as there will likely be name collisions when searching

**Comment by kosack** [comment]
*configurator* is also an obvious one, and not gimmicky as it's a dictionary word for something that configures

**Comment by kbruegge** [comment]
I agree that its easier to find if the name is unique and "googleable". I don't mind creative names if they're unique. 
Of the three options "configeater" seems to be the most unique according to a quick google search. I personally like "configurist" but so do many other people as well apparently. I guess CTA configurist should still be findable  👀
